### ISSUE TITLE
Clarification on the Spec

### ISSUE THREAD

**Issue by rbarden** [traindown/traindown#3]
Hey there, I'm building a PHP lexer/parser and am almost ready to share it with you and the world. However, I've come across some inconsistent behavior that I think needs to be reconciled.

### The Spec
> From a LOAD token until EOF or another LOAD token.

However, the guide says LOAD, REP, SET, and FAIL tokens can appear in any order per performance. This is not actually the case though. For the first performance, they _can_ appear in any order, but then for subsequent performances the LOAD has to appear first.

### Example 1
```
Squat:
  5r 2s 500 550 10r 3s
```
yields two performances
 * 2 sets of 5 reps with a 500 load
 * 3 sets of 10 reps with a 550 load

### Example 2 
```
Squat:
5r 2s 500 10r 3s 550
```
yields two performances 
 * 3 sets of 10 reps with a 500 load
 * 1 set of 1 rep with a 550 load

I think there should be consistency between initial and subsequent performances and there are two options:

1. LOAD always has to come first. (changing the initial performance behavior)
2. Those four tokens can appear in any order, but repeat token types start a new performance. (in the second example above: the `10r` would start a new performance instead of overwriting the first performance rep count).

Thoughts?

**Comment by scottt2** [comment]
@rbarden You are :100: correct. That is a mistake in the spec and one I am thankful you caught. The "any order" was a feature I added for 1.2 and obviously I did not have enough rigorous testing of it.

All prior versions of Traindown had 
> LOAD always has to come first. (changing the initial performance behavior)
as a constraint. I wanted to allow for more flexibility but obviously I didn't kick the tires enough.

I absolutely love your addition of
> Those four tokens can appear in any order, but repeat token types start a new performance
to the spec. With your permission, I would like to add it. It is a very straightforward, simple, and covers any use cases I can think off the top of my head.
